wasm/test_page/bergamot.html

```diff
@@ -156,7 +156,7 @@
       console.debug("Creating TranslationModel with config:", modelConfigWithoutModelAndShortList);
       translationModel = new Module.TranslationModel(modelConfigWithoutModelAndShortList, alignedModelMemory, alignedShortlistMemory);
     } catch (error) {
-      console.error(error);
+      log(error);
```

Comment:
Not console.log?

Reply:
Nopes. Otherwise it log the failure silently and you will see nothing on the actual page 👍